apps/molgenis-components/src/components/tables/TableMolgenis.vue

```diff
@@ -19,7 +19,7 @@
             class="mb-0 align-text-bottom text-nowrap"
             @click="onColumnClick(col)"
           >
-            {{ col.label }}
+            {{ col.label || col.name || col.id }}
```

Comment:
i thought the label was set the the name in case of empty label, and id would only be used internal for linking ( not ui )

Reply:
indeed, in the new frontend there should ALWAYS be a label.

Reply:
Ah, I now suddenly see what is needed => in schema editor this label should be produced before the preview of the form can be shown. The code for that is missing something.

Reply:
So the schema editor should create a default label in case it is not entered, or should it be marked as required? In both cases, should there be a db update that adds a label to columns without one?

Reply:
I have added another commit that I think fixes it in the right place (untested). If you agree you can undo this commit and merge.

Reply:
I agree with your fix, but how about already existing columns and uploaded schema's?